tests/unit/test_session.py

```diff
@@ -725,23 +724,10 @@ def unit_of_work(txn, *args, **kw):
         self.assertEqual(args, ())
         self.assertEqual(kw, {})
 
-        expected_options = TransactionOptions(read_write=TransactionOptions.ReadWrite())
-        gax_api.begin_transaction.assert_called_once_with(
-            session=self.SESSION_NAME,
-            options=expected_options,
-            metadata=[("google-cloud-resource-prefix", database.name)],
-        )
-        gax_api.rollback.assert_called_once_with(
```

Comment:
Can you confirm that we shouldn't be expecting a rollback here? If we aren't, perhaps add an assertion that there isn't one and a comment explaining why.

Reply:
Done